### PR TITLE
feat(pattern/radio): add input radio pattern

### PR DESCRIPTION
**Pull request by gael-boyenval**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Comment by gael-boyenval** [comment]
@melanieanglardLMFR : a quick review here, nothing bad, could be merged as it : 

https://github.com/adeo/mozaic-design-system/pull/297

File: src/pages/Components/Radio/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Focus
+
+When the radio is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
+
+<preview path="src/pages/Components/Radio/previews/radio-focus-state" nude="true"></preview>
+
+### Disabled
+When the radio field is inactive or inaccessible. It is usually used when the input, for some reason, is forbidden or blocked to be edited.
+
+<preview path="src/pages/Components/Radio/previews/radio-disabled-state" nude="true"></preview>
```
Comment (caqueste) [inline src/pages/Components/Radio/index.md]: There is not an outline on the disable state.
Also, can we have a disable cursor on the hover?

File: src/pages/Components/Radio/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'Radio'
+order: 8
+---
+
+> Checkboxes are used to **select only one option in a list**. They usually find their place in forms.
```
Comment (gael-boyenval) [inline src/pages/Components/Radio/index.md]: Radio, not Checkboxes

File: src/pages/Components/Radio/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### Hover
+
+When the radio is hovered due to user navigation.
+
+<preview path="src/pages/Components/Radio/previews/radio-hover-state" nude="true"></preview>
+
+### Focus
+
+When the radio is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
```
Comment (gael-boyenval) [inline src/pages/Components/Radio/index.md]: not sure the radio get a focus when clicked can you check that ?

**Comment by caqueste** [comment]
I think *Focus*, *Hover*, *Valid* and *Invalid* states should be in the section **Code**

File: src/pages/Components/Radio/index.md (hunk 2 above)
Comment (caqueste) [reply src/pages/Components/Radio/index.md]: To complete the definition of a radio :
Radios are components that, unlike checkboxes, can't be used alone.
They are used when you have to make a single choice in a list, that includes minimum two options.

Comment (gael-boyenval) [reply src/pages/Components/Radio/index.md]: "Used to select a single option from a list". I think it's saying the same thing in fewer words... 
I think the "They usually find their place in forms." is unnecessary.

**Pull request merged**